Multiple languages ​​for the narrative web and optional other additions

SNoiraud opened the pull request:
Add the possibility to generate the narrative web for 6 languages max.
Add an optional option to show object tags
Add optional toggle for sections.
Add a scroll to top button

The web calendar doesn't work when multiple languages is in use and we link people to the narrative web.
after multiple tries without succes, I integrate webcal in the narrative web.
For each language, I use the related holidays calendar. (#07151)

Fixes #11666

SNoiraud commented:
Here is a video showing all these new functionalities.
As I can't add video, I renamed the file .txt.
[PR1051.mp4.txt](https://github.com/gramps-project/gramps/files/4829840/PR1051.mp4.txt)

wroldwiedbwe commented:
👍 Really good. Thanx

![PR1051-screenshot-of feature](https://user-images.githubusercontent.com/29416263/85687351-24a14880-b714-11ea-8a34-ddc7ed1ba8b5.png)

SNoiraud commented:
I added a go to top button.
Here is a video showing this new functionality.
As I can't add video, I renamed the file .txt.
[gototop.mp4.txt](https://github.com/gramps-project/gramps/files/5294912/gototop.mp4.txt)
